sdg_mapping/sdg_index/sdg_index_utils.py

```diff
@@ -0,0 +1,113 @@
+import numpy as np
+import pandas as pd
+
+import sys
+import os
+import json
+
+from sdg_mapping import project_dir
+
+with open(f'{project_dir}/data/aux/sdg_index_data_opts.json', 'r') as f:
+    state_dict = json.load(f)
+
+
+def sdg_index_file_path(year):
+    """sdg_index_file_path
+    To fetch filepath of SDG Index datasets.
+
+    Args:
+        year (str): Year of SDG Index datasets
+
+    Returns:
+        (str): File path
+
+    """
+    fname = f'{year}_sdg_index.xlsx'
+    return f'{project_dir}/data/raw/{fname}'
+
+
+def read_workbook(data_path, state):
+    """read_workbook
+    To read an Excel read_workbook
+
+    Args:
+        data_path (str): File path of an SDG Index dataset
+        state (str): State of dataset needed; raw or data
+
+    Returns:
+        (pd.DataFrame): Parsed SDG Index data
+
+    """
+    if state == 'data':
+        df = pd.read_excel(open(data_path, 'rb'),
+                  sheet_name=state_dict[state],header = 1)
+
+    elif state == 'raw':
+        df = pd.read_excel(open(data_path, 'rb'),
+                  sheet_name=state_dict[state])
+    return df
+
+def parse_2019_sdg_index(dataset, state):

```

Comment:
```suggestion
def parse_2019_sdg_indicator(dataset):
```

It doesn't look like `state` is used in this function as it's only for the indicator data, not the raw data. Also suggest renaming it to make that clearer.